inc/front/cdn.php

```diff
@@ -1,6 +1,11 @@
 <?php
 defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
 
+// Nothing to do if XMLRPC request.
+if ( defined( 'XMLRPC_REQUEST' ) ) {
+	return;
```

Comment:
Doing a return statement in the global scope is bad practice.

I suggest to wrap the potentially problematic hooks calls in a conditional instead